jina/resources/extra-requirements.txt

```
@@ -26,7 +26,7 @@
 # demo: required for the hello world demos
 
 numpy:                      core
-protobuf>=3.19.1,<=3.20.1:  core
+protobuf>=3.21.0:           core
```

https://pypi.org/project/protobuf/4.21.0/

```suggestion
protobuf>=4.21.0:           core
```